src/amb_kf.c

```diff
@@ -531,7 +531,7 @@ s32 find_index_of_element_in_u8s(const u32 num_elements, const u8 x, const u8 *l
       return i;
     }
   }
-  return -1;
+  assert(false);
```

Comment:
maybe change the name and add comment to indicate it should only be used on permuted prn lists, keep assert(false)?